docs/releases.md

```diff
@@ -0,0 +1,25 @@
+# Tobira versions
+
+(For context, please see our [versioning policy](./versioning-policy.md)!)
+
+This document lists all Tobira releases with the kinds of breaking changes done in each release.
+For the meaning of the columns, see the section "breaking changes" in the versioning policy.
+The symbol 🔴 means that a breaking change of that kind happened in that release.
+For columns that only track the required version of external software, the required version number is also listed.
+
+(NOTE: these releases are dummy releases!)
+
+| Release              | Date       | *config* | *auth* | *cli* | *meili*    | *pg* | *tmod* | *ocversion* | *ocreq* |
```

Comment:
I have no idea what “tmod” means. Don't shorten these.

Reply:
My initial worry was that the table would exceed page width. As this is not the case, you're right.